### DISCUSSION TITLE
Saving meshes *solved*

### DISCUSSION THREAD

**Discussion by pitchbent** [cryinkfly/Autodesk-Fusion-360-for-Linux#229] [Q&A] [answered]
As the title suggests, I am wondering how to save meshes locally. The wine explorer shows the file but it does not appear on my systems file explorer (using manjaro).
I think the documents folder should link to my actual documents folder but it is not working.

Does anyone now a fix for this? Thanks in advance!

**Comment by cryinkfly** [comment, marked as answer]
Here is the reason why its so now! :+1: 

- https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/issues/218
